src/components/Input/InputPair/index.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import { ReactNode } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition, library, icon } from "@fortawesome/fontawesome-svg-core";
+import styles from "./inputpair.module.css";
+
+type InputPairProps = {
+	icon?: IconDefinition;
+	className?: string;
+	children: [ReactNode, ReactNode];
+};
+
+export default function InputPair({ className, ...props }: InputPairProps) {
```

Comment:
This can be replaced with just `{...props}` then recollected in the body as `const {className, icon} = props`

Reply:
Noted